frontend/TestRun/Jenkins/SCTParameterWizard.svelte

```diff
@@ -0,0 +1,414 @@
+                    onChange: function (e, params) {
+                        params.scylla_version = "";
+                        params.scylla_repo = "";
+                    }
+                },
+                updateDbPackages: {
+                    name: "Package Update URI",
+                    type: StringParam,
+                    internalName: "update_db_packages",
+                    condition: (params, defs) => (true),
+                    onChange: function (e, params) {
+                        //empty
+                    }
+                },
+                newScyllaRepo: {
+                    name: "Repository to pull newest scylla version from",
+                    type: StringParam,
+                    internalName: "new_scylla_repo",
+                    condition: (params, defs) => (defs.scyllaVersion.currentSource == "rolling_upgrade"),
+                    onChange: function (e, params) {
+                        //empty
+                    }
+                },
+                baseVersions: {
+                    name: "Base Versions to upgrade from",
+                    type: StringParam,
+                    internalName: "base_versions",
+                    condition: (params, defs) => (defs.scyllaVersion.currentSource == "rolling_upgrade"),
+                    onChange: function (e, params) {
+                        //empty
+                    }
+                },
+            }
+        },
+        testConfiguration: {
+            name: "Test configuration",
+            params: {
+                testName: {
+                    name: "Test Name",
+                    type: StringParam,
+                    internalName: "test_name",
+                    condition: (params, defs) => true,
+                    onChange: function (e, params) {
+                        //empty
+                    }
+                },
+                testConfig: {
+                    name: "Configuration to use",
+                    type: StringParam,
+                    internalName: "test_config",
+                    condition: (params, defs) => true,
+                    onChange: function (e, params) {
+                        //empty
+                    }
+                },
+                email_recipients: {
+                    name: "Email Report Recepients",
+                    description: "Comma separated list of email recipients",
+                    type: StringParam,
+                    internalName: "email_recipients",
+                    condition: (params, defs) => true,
+                    onChange: function (e, params) {
+                        //empty
+                    }
+                },
+                stressDuration: {
+                    name: "Custom Stress Command Duration",
+                    type: StringParam,
+                    internalName: "stress_duration",
```

Comment:
we need to have the internalName visible on the form, it would be confusing with it